### ISSUE TITLE
Got error The requested range is not satisfiable

### ISSUE THREAD

**Issue by kenorb** [contabo/cntb#37]
I was trying to upload 500G file to the bucket (which took around 2 days straight), but it failed for unknown reason.

```
$ cntb create object --storageId xxx-yyy --bucket mybucket --prefix prefix/ --path myfile
FATAL Could not create object file prefix/myfile. Got error The requested range is not satisfiable
```

I'll try again, but if it fails again, then this tool isn't appropriate for uploading the big files to the Contabo bucket without ability to retry from were it stopped (waste of resources).

**Comment by kenorb** [comment]
Second try, after 25h of uploading, it failed again:

> FATAL Could not create object file. Got error The requested range is not satisfiable.

**Comment by andreiduca03** [comment]
Hi, the issue is fixed now in the new version: v1.4.9

**Comment by kenorb** [comment]
I've tried v1.4.9 version, now there is a different error on upload after 4.5h of upload:

> FATAL Could not create object file myfile.tar. Got error {"message":"failure to get a peer from the ring-balancer"}

**Comment by andreiduca03** [comment]
we added a fix on version [v1.4.10](https://github.com/contabo/cntb/commit/113a5ea28903fe9a4876953ac96c068292c106fd), please let us know if there are any issues with the file upload
